compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -23,6 +23,7 @@
 #include "llvm/Bitcode/BitcodeWriter.h"
 #include "llvm/Support/Signals.h"
 
+#include <cassert>
```

Comment:
This doesn't seem to be necessary. (Maybe your C++ tools added it automatically?)